### PR TITLE
Mathematica package

### PR DESCRIPTION
**Pull request by schlyfts**
Dummy package for Mathematica

### CODE REVIEW

File: var/spack/repos/builtin/packages/mathematica/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from spack import *
+
+import spack.environment
+
+class Mathematica(Package):
+    """
+    Mock package for Mathematica
+    """
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/mathematica/package.py]: @schlyfts Can you add a description ? `__doc__` is actually used for module file creation and that's what a user will see if he'll type `module help mathematica`. I think I'll need to fix `intel` and `intelmpi` in the same way...

**Comment by schlyfts** [comment]
Documentation added

**Comment by alalazo** [comment]
If you're done I'll merge this.

**Comment by schlyfts** [comment]
Merge and be damned ;-)

**Pull request merged**